pym/bob/utils.py

```diff
@@ -215,7 +229,7 @@ def canSymlink():
     # it is possible to create symlinks. Try to create a symlink to see if we
     # have the privilege. Either the symlink() call fails directly or MSYS
     # silently creates a copy (unless MSYS=winsymlinks:nativestrict is set).
-    if sys.platform in ('msys', 'cygwin', 'win32'):
+    if isWindows():
```

Comment:
Either you revert this particular change too or you adapt the failing unit tests by mocking `isWindows()` in them.

Reply:
reverted.